app/containers/TwoFactor/styles.js

```diff
@@ -32,7 +34,8 @@ export default StyleSheet.create({
 		...sharedStyles.textRegular
 	},
 	button: {
-		marginBottom: 0
+		marginBottom: 0,
+		width: '48%'
```

Comment:
Looks weird, I guess the button don't need to cover the entire modal, this is a pattern also used on our UploadModal.
WDYT @diegolmello?

Reply:
Right.
I thought this PR would be only for the title btw.

Reply:
No problem!

Reply:
Done!